plugin/pkg/admission/resourcequota/controller.go

```diff
@@ -105,7 +105,7 @@ func NewQuotaEvaluator(quotaAccessor QuotaAccessor, registry quota.Registry, loc
 
 		registry: registry,
 
-		queue:      workqueue.New(),
+		queue:      workqueue.NewNamed("admission_quota_controller"),
```

Comment:
Ok. I think what is throwing me off is `controller_serviceaccount_tokens_service` and `controller_serviceaccount_tokens_secret` which both start with `controller`, but I guess that's the "area", whereas in `admission_quota_controller`, the "area" is "admission".